xmtp_mls/src/groups/members.rs

```diff
@@ -40,14 +40,25 @@ impl MlsGroup {
             .members
             .into_iter()
             .map(|(inbox_id, sequence_id)| (inbox_id, sequence_id as i64))
+            .filter(|(_, sequence_id)| *sequence_id != 0) // Skip the initial state
```

Comment:
The original behavior is as follows:

1. When a group is created, the metadata will read `(creator_inbox_id, 0)`. So a sequence_id of 0
2. The old logic would return no association state, and therefore an empty member list

I'm not sure if this is by design, or a side effect. Either way, adding this filter preserves that original behavior, while ensuring that the new error handling logic on L50 doesn't trigger in this case.